central/clusters/deployer.go

```diff
@@ -70,8 +70,9 @@ func MakeClusterImageNames(flavor *defaults.ImageFlavor, c *storage.Cluster) (*s
 
 // deriveScannerRemoteFromMain sets scanner-slim image remote, so that it comes from the same location as the main image
 func deriveScannerRemoteFromMain(mainImage *storage.ImageName, metaValues *charts.MetaValues) {
-	scannerSlimRemote := strings.Replace(mainImage.Remote, "/main", "/"+metaValues.ScannerSlimImageRemote, 1)
-	metaValues.ScannerSlimImageRemote = scannerSlimRemote
+	scannerRemoteSlice := strings.Split(mainImage.Remote, "/")
+	scannerRemoteSlice[len(scannerRemoteSlice)-1] = metaValues.ScannerSlimImageRemote
```

Comment:
There is a risk of panic if `len(scannerRemoteSlice) == 0`. There are probably other problems in this path if `mainImage` is not set. But perhaps we should be defensive here and avoid panics. 

```go
if len(scannerRemoteSlice) > 0 {
      // update ScannerSlimImageRemote
}
```

Reply:
I read the docs of `strings.Split` and having a 0-lenght silce is possible only when both arguments are empty - i.e., `strings.Split("","")`.
But let's be on the safe side - I'll add the extra check.

Reply:
Got it in [d262ee7](https://github.com/stackrox/stackrox/pull/5948/commits/d262ee7f187fdb301b2802607c05c9e760651b4c)